docs/static/spec/rasa.yml

```diff
@@ -546,22 +549,48 @@ paths:
       - Model
       summary: Perform an intent evaluation
       description: >-
-        Evaluates intents against the currently loaded Rasa model or the model specified in the query.
+        Evaluates NLU model against a model or using cross-validation.
       parameters:
       - $ref: '#/components/parameters/model'
+      - $ref: '#/components/parameters/callback_url'
+      - in: query
+        name: cross_validation_folds
+        schema:
+          type: integer
+          default: null
+          description: >-
+            Number of cross validation folds. If this parameter is specified
+            the given training data will be used for a cross-validation instead of
+            using it as test set for the specified model. Note that this is only
+            supported for YAML data.
       requestBody:
         required: true
         content:
-          text/yml:
+          application/x-yaml:
             schema:
-              $ref: '#/components/schemas/NLUTrainingData'
+              type: string
+              description: >-
+                NLU training data and model configuration. The model configuration is
```

Comment:
So the config and nlu data are provided together in one file?

Reply:
yep, one long yaml string. Since 2.0 everything can be in one file so we don't really separate anymore.